packages/0x.js/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "0.36.1",
+        "changes": [
+            {
+                "note": "Remove truffle-hdwallet from artifact tests",
```

Comment:
CHANGELOG's should actually only include changes that are relevant to users of our libraries. Usually things impacting the public interface or behavior of the library. Think this one doesn't qualify. Thoughts?

Reply:
Yeah exactly... "Updated dependencies" is not ideal or even truthful in this case... I could be open to changing our conventions here, thoughts @LogvinovLeon?

Reply:
I'm happy to change it to signal that something happened internally that is not relevant publicly. Also dropping the PR number?